### PR TITLE
HDDS-4904. Remove mention of CSI support

### PR DESCRIPTION
**Pull request by elek**
## What changes were proposed in this pull request?

Thanks for @arp7 for the suggestion to document better the current state of Ozone CSI driver and the guarantees what it provides.

Patch includes some additional warning and changing the title image with adding alpha tag to the CSI/K8s icon.

Please note that there is an ongoind discussion about this approach on the Jira and on the mailing list. I created the PR only to show my proposal, do not merge it before any consensus.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4904

## How was this patch tested?

`hugo server` in `hadoop-hdds/docs`

### CODE REVIEW

**Review by arp7 — APPROVED** [review]

**Comment by arp7** [comment]
I am +1 on this. I will hold off as you requested, please feel free to commit though. Thank you!

**Review by dineshchitlangia — APPROVED** [review]
+1 LGTM

File: hadoop-hdds/docs/content/interface/CSI.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ summary: Ozone supports Container Storage Interface(CSI) protocol. You can use O
   limitations under the License.
 -->
 
+<div class="alert alert-warning" role="alert">
+
+Ozone CSI support still in alpha phase and buckets can be mounted only via 3rd party S3 compatible Fuse implementation (like Goofys). Fuse over S3 can provide only limited performance compared to a native Fuse file systems. Long-term Ozone may support a custom solution to mount buckets which provides better user experience (with fuse or NFS or any other solution). Until that CSI is recommended to use only if you can live with this limitation and your use case is tested carefully.
+</div>
```
Comment (dineshchitlangia) [inline hadoop-hdds/docs/content/interface/CSI.md:30]:
```suggestion
Ozone CSI support is still in alpha phase and buckets can be mounted only via 3rd party S3 compatible Fuse implementation (like Goofys). Fuse over S3 can provide only limited performance compared to a native Fuse file system. Long-term Ozone may support a custom solution to mount buckets which provides better user experience (with fuse or NFS or any other solution). Until that CSI is recommended to use only if you can live with this limitation and your use case is tested carefully.
</div>
```

**Review by dineshchitlangia — CHANGES_REQUESTED** [review]
Sorry for NIT :)

**Comment by elek** [comment]
@dineshchitlangia NIT is fixed. I assume we are good to go, are we? (note I also added 436fc0c)

**Review by dineshchitlangia — APPROVED** [review]
+1 @elek , all clear now :)

**Comment by elek** [comment]
Merging it. Thanks the review @arp7 and @dineshchitlangia

**Pull request merged**